src/vmshepherd/presets/preset.py

```diff
@@ -10,6 +10,7 @@ class Preset:
     def __init__(self, name: str):
         self.iaas = None
         self.healthcheck = None
+        self.unmanaged = False
```

Comment:
I presumed that "unmanaged" is a runtime state, that could be switch on/off via API, trigger by events (unavailability of some part)...

Reply:
yep, maybe, but this is only implementation of unmanaged fuctionality in static configuration
- preset configuration or config file of vmshepherd

Reply:
its required  for AWS where presets are managed by ASG.